java/src/main/java/com/google/cloud/dataproc/templates/jdbc/JDBCToBigQuery.java

```diff
@@ -157,4 +131,35 @@ public void runTemplate() {
         .option("table", bqLocation)
         .save();
   }
+  
+  void validateInput() {
```

Comment:
We need to put more comprehensive checks using jakarta validator framework.
Refer to JdbcToGCSConfig for examples.